modules/mobile_connection.lua

```diff
@@ -8,9 +8,26 @@ local module = {
 function module.MobileConnection(connection)
   res = { }
   res.connection = connection
+
+  res.sessions = {}
```

Comment:
@VVeremjova this part of refactoring is not performed yes as I remember ( mobile_session still split messages for sessions). Please remove this changes because it makes developer think that sessions are spited in connection ,